templates/controllers/grid/issues/form/issueForm.tpl

```diff
@@ -43,16 +43,15 @@
 		{assign var=issuePublished value=false}
 	{/if}
 
-	{if $issuePublished}
 		{fbvFormArea id="datePublishedArea" title="editor.issues.datePublished"}
 			{fbvFormSection}
 				{if $issuePublished}
 					{fbvElement type="text" id="datePublished" value=$datePublished size=$fbvStyles.size.SMALL class="datepicker"}
+				{else}
+					{fbvElement type="text" id="datePublished" value=$datePublished size=$fbvStyles.size.SMALL class="datepicker" label="editor.issues.datePublished.notPublished.description"}
```

Comment:
On my browser I am getting something like:

![image](https://github.com/pkp/ojs/assets/560456/acbae7b1-4034-4b5a-aa6a-664b2b6e80b2)

Not really important but perhaps by adding a simple `autocomplete="off"` there will do the trick.

Reply:
check